plugins/thunderbolt/fu-thunderbolt-device.c

```diff
@@ -444,6 +444,28 @@ fu_thunderbolt_device_write_firmware(FuDevice *device,
 	return TRUE;
 }
 
+static gboolean
+fu_thunderbolt_device_probe(FuDevice *device, GError **error)
+{
+	g_autoptr(FuUdevDevice) udev_parent = NULL;
+
+	/* FuUdevDevice->probe */
+	if (!FU_DEVICE_CLASS(fu_thunderbolt_device_parent_class)->probe(device, error))
+		return FALSE;
+
+	/* if the PCI ID is Intel then it's signed, no idea otherwise */
+	udev_parent = fu_udev_device_get_parent_with_subsystem(FU_UDEV_DEVICE(device), "pci");
+	if (udev_parent != NULL) {
+		if (!fu_device_probe(FU_DEVICE(udev_parent), error))
+			return FALSE;
+		if (fu_udev_device_get_vendor(udev_parent) == 0x8086)
```

Comment:
8087 too, no?

Reply:
I've not seen that for tbt controllers, but I can certainly add it if you think it's likely.

Reply:
8087 is for USB devices only apparently: https://pci-ids.ucw.cz/read/PC/8087